app/database/models/server/thread.ts

```diff
@@ -54,6 +54,9 @@ export default class ThreadModel extends Model {
     /** loaded_in_global_threads : Flag to differentiate the unread threads loaded for showing unread counts/mentions */
     @field('loaded_in_global_threads') loadedInGlobalThreads!: boolean;
 
+    /** viewed_at : The timestamp showing when the user's last opened this thread (this is used for the new line message indicator) */
+    @field('viewed_at') viewedAt!: number;
```

Comment:
I'm going to allow it unless someone else says otherwise, but this should be it's own PR